src/features/glsllintProvider.ts

```diff
@@ -198,6 +198,14 @@ export class GLSLLintingProvider {
       '.task': 'task', // for a task shader
     };
 
+    // glslangValidator supports compound extensions now that it supports multiple 
+    // shader languages:
+    // .glsl   for .vert.glsl, .tesc.glsl, ..., .comp.glsl compound suffixes
+    // .hlsl   for .vert.hlsl, .tesc.hlsl, ..., .comp.hlsl compound suffixes
+    if (fileName.endsWith(".glsl") || fileName.endsWith(".hlsl")) {
```

Comment:
Please respect code format: use single quotes for string literals and remove trailing whitespaces.